### PR TITLE
refactor: using useSpace()

### PR DESCRIPTION
**Pull request by rajsam003**
This PR replaces the use whole number with `useSpace()` values for `padding`, `marging`, `width` and `height`

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

File: src/app/screens/HomeTabs/Albums/AlbumArtworks.tsx [hunk 1]
```diff
@@ -52,20 +53,28 @@ export const AlbumArtworks = () => {
         label={album.name}
         rightElements={
           <Flex flexDirection="row" alignItems="center">
-            <Touchable onPress={deleteAlbumHandler} style={{ marginRight: 20 }}>
-              <TrashIcon fill="black100" width={25} height={25} />
+            <Touchable onPress={deleteAlbumHandler} style={{ marginRight: space(2) }}>
+              <TrashIcon
+                fill="black100"
+                width={space(2) + space(0.5)}
```
Comment (patrinoua) [inline src/app/screens/HomeTabs/Albums/AlbumArtworks.tsx:59]: I am not a big fan of this

Comment (patrinoua) [reply src/app/screens/HomeTabs/Albums/AlbumArtworks.tsx:59]: maybe we can only use space when it comes to paddings and margins, where we have standard "10", "20", etc, and not when we are describing the size of something.
What do you think?

Comment (rajsam003) [reply src/app/screens/HomeTabs/Albums/AlbumArtworks.tsx:59]: Yes, I was thinking the same

Comment (rajsam003) [reply src/app/screens/HomeTabs/Albums/AlbumArtworks.tsx:59]: Ok, Lemme revert only for width and heights  😄

Comment (patrinoua) [reply src/app/screens/HomeTabs/Albums/AlbumArtworks.tsx]: perfect! 🚀

**Pull request merged**